Tests/Fakes/FakeCellNibView.swift

```diff
@@ -44,6 +44,16 @@ struct FakeCellNibViewModel: CellViewModel {
         self.expectationDidSelect?.fulfillAndLog()
     }
 
+    init(
+        id: UniqueIdentifier = String.random,
+        expectationConfigureCell: XCTestExpectation? = nil,
+        expectationDidSelect: XCTestExpectation? = nil
+    ) {
```

Comment:
Doesn't the compiler synthesize this `init` for us for free?

If possible, it would be great to avoid having to manually write all these `init`s

Reply:
If we want to use the init method implemented by the compiler, we need to change `let id` to `var id`.

Reply:
```swift
// Version A

struct FakeCellNibViewModel: CellViewModel {
    var id: UniqueIdentifier = String.random
}

let viewModel1 = FakeCellNibViewModel() // String.random, keep calling `FakeCellNibViewModel()` to init a new viewModel

var viewModel2 = FakeCellNibViewModel()
viewModel2.id = "1" // "1"
```

```swift
// Version B

struct FakeCellNibViewModel: CellViewModel {
    let id: UniqueIdentifier // remove default value, always call `FakeCellNibViewModel(id:)` to init a new viewModel
}

let viewModel1 = FakeCellNibViewModel(id: String.random) // String.random
let viewModel2 = FakeCellNibViewModel(id: "1") // "1"
```

Reply:
let's leave as-is!